markdown.dtx

```diff
@@ -25557,6 +25944,11 @@ function M.new(options)
     table.insert(extensions, jekyll_data_extension)
   end
 
+  if options.lineBlocks then
+    local line_block_extension = M.extensions.line_blocks()
+    table.insert(extensions, line_block_extension)
+  end
+
```

Comment:
The order in which extensions are applied is significant and may affect correctness (even though it is fine here).